src/pybind/mgr/dashboard/frontend/e2e/block/images.e2e-spec.ts

```diff
@@ -30,4 +32,37 @@ describe('Images page', () => {
       expect(images.getTabText(2)).toEqual('Overall Performance');
     });
   });
+
+  describe('create, edit, & delete image test', () => {
+    beforeAll(() => {
```

Comment:
Since you are using the same string multiple times, I would suggest to extract it into a var and reuse it.
You can pick better names, but I would suggest we make them descriptive of where they are being use, in this case in the `e2e images`:
```suggestion
    const poolName = 'e2e_images_pool';
    const imageName = 'e2e_images_image';
    const newImageName = 'e2e_images_image_new';
    
    beforeAll(() => {
```

Reply:
+1 for extracting it into a var.